conda_forge_tick/migrators/jpegturbo.py

```diff
@@ -0,0 +1,27 @@
+from conda_forge_tick.migrators.core import MiniMigrator
+import os
+
+
+def _parse_jpeg(lines):
+    new_lines = []
+    for line in lines:
+        line = line.replace(" jpeg\n", " libjpeg-turbo\n")
+        line = line.replace(" jpeg", " libjpeg-turbo ")
```

Comment:
I think these lines address the concern with `openjpeg`

i am trying to find a "complicated" recipe to test against. So openjpeg is a good thing to include in the final test.

Though maybe stepping stone is required.